### ISSUE TITLE
Unable to build openshift installer using the latest master.

### ISSUE THREAD

**Issue by ScrapCodes** [openshift/installer#3576]
Using the current latest master: https://github.com/openshift/installer/commit/d3332b3b54dfcdebb674a7c5cdc36c9646da663c

# Version

```console
$ openshift-install version
<your output here>
```

# Platform:
RHEL 7.8

# What happened?

Output of hack/build.sh

```
hack/build.sh 
+ minimum_go_version=1.13
++ cut -d ' ' -f 3
++ go version
+ current_go_version=go1.13.6
++ version 1.13.6
++ IFS=.
++ printf '%03d%03d%03d\n' 1 13 6
++ unset IFS
++ version 1.13
++ IFS=.
++ printf '%03d%03d%03d\n' 1 13
++ unset IFS
+ '[' 001013006 -lt 001013000 ']'
+ MODE=release
++ git rev-parse --verify 'HEAD^{commit}'
+ GIT_COMMIT=d3332b3b54dfcdebb674a7c5cdc36c9646da663c
++ git describe --always --abbrev=40 --dirty
+ GIT_TAG=unreleased-master-3021-gd3332b3b54dfcdebb674a7c5cdc36c9646da663c
+ GOFLAGS=-mod=vendor
+ LDFLAGS=' -X github.com/openshift/installer/pkg/version.Raw=unreleased-master-3021-gd3332b3b54dfcdebb674a7c5cdc36c9646da663c -X github.com/openshift/installer/pkg/version.Commit=d3332b3b54dfcdebb674a7c5cdc36c9646da663c'
+ TAGS=
+ OUTPUT=bin/openshift-install
+ export CGO_ENABLED=0
+ CGO_ENABLED=0
+ case "${MODE}" in
+ LDFLAGS=' -X github.com/openshift/installer/pkg/version.Raw=unreleased-master-3021-gd3332b3b54dfcdebb674a7c5cdc36c9646da663c -X github.com/openshift/installer/pkg/version.Commit=d3332b3b54dfcdebb674a7c5cdc36c9646da663c -s -w'
+ TAGS=' release'
+ test '' '!=' y
+ go generate ./data
go: github.com/Azure/go-autorest/autorest/adal@v0.8.1-0.20191028180845-3492b2aff503: invalid pseudo-version: git fetch --unshallow -f https://github.com/Azure/go-autorest in /root/go/pkg/mod/cache/vcs/630f3be74475b7a26ecdaff7267660f1df0d219fbc2c8341c8fb6faf3f2ca731: exit status 128:
	fatal: git fetch-pack: expected shallow list
```
# What you expected to happen?

Enter text here.

# How to reproduce it (as minimally and precisely as possible)?
Start with a fresh clone to reproduce.

```console
$ hack/build.sh
```

# Anything else we need to know?

Enter text here.

# References

- enter text here.

**Comment by ScrapCodes** [comment]
Please start with a fresh clone to reproduce.

**Comment by abhinavdahiya** [comment]
Maybe you have a corrupted go mod cache?

Also maybe you have some go mod force environment set, as the build should be using the vendor directory and not fetching modules.

**Comment by ScrapCodes** [comment]
I could not fix this, but changing to release-4.4 branch seemed to work for me.
